### ISSUE TITLE
Blueberry Applet : Please add features

### ISSUE THREAD

**Issue by Hilyxx** [linuxmint/blueberry#72]
Hello

Blueberry applet features aren't as complete as the old bluetooth applet in linux mint 16 or 17. 

Is it possible to add more features : switch for disconnect bluetooth, a submenu for the paired devices ... ?

I made this mockup for more precision. 
![capture d ecran du 2018-12-07 07 17 30](https://user-images.githubusercontent.com/21991448/49631804-be4ebb00-f9f3-11e8-9243-3b848d528851.png)

Thank you :)

**Comment by monokles** [comment]
yeah, I implemented a more basic version of that menu you mentioned, however my PR does not include a toggle button for bluetooth itself. This is something I very much want myself as well since opening blueberry itself is slow, and uses way too many system resources, so simply controlling it from the dropdown would be ideal.
I plan to add that later at some point, but I'm very busy myself, so will need to find a moment to sit down for it. And frankly I don't know if the maintainers are even interested in merging this... Considering I haven't received a reaction on my previous PR at all.

**Comment by kushraj** [comment]
Hi @monokles , will it be possible to recompile it with libapp-inidicator support somehow ? so far its the only tray protocol sway support afaik.
